models/newsModels.js

```diff
@@ -5,3 +5,11 @@ exports.fetchTopics = () => {
     return rows;
   });
 };
+
+exports.fetchArticle = (id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id=$1;`, [id])
+    .then(({ rows }) => {
+      return rows[0];
+    });
```

Comment:
There is currently no error handling for this endpoint. How would your server reply if there was an id number that was valid, but but no article exists with that id number? Or how would your server respond if the provided id number was of an invalid data type - like /api/articles/cheese for example?

Reply:
added error handling and changed the name of the function could you have a look please 
thank you